pkg/network/metrics/pprof.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"net/http"
+	"net/http/pprof"
+)
+// PprofService https://golang.org/pkg/net/http/pprof/.
+type PprofService Service
+
+// NewPprofService created new service for gathering pprof metrics.
+func NewPprofService(cfg Config) *Service {
+	handler := http.NewServeMux()
+	handler.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
BTW, do we really need to push it down here if that's the only service bound to this instance of `http`? Maybe move it to the `/` and be done with it?